src/libraries/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -587,7 +587,7 @@ public void SetWindowSize_GetWindowSize_ReturnsExpected()
     }
 
     [Fact]
-    [PlatformSpecific(TestPlatforms.AnyUnix)]
+    [PlatformSpecific(TestPlatforms.Browser | TestPlatforms.iOS | TestPlatforms.MacCatalyst | TestPlatforms.tvOS)]
```

Comment:
All the tests that you have modified verify that the method throws when it's not supported or the input is invalid.

Please enable `SetWindowSize_GetWindowSize_ReturnsExpected` test for configs which actually support the new API.

https://github.com/dotnet/runtime/blob/8d6b52f2b58bad7b1c48d98305fbcc9dc6f56921/src/libraries/System.Console/tests/WindowAndCursorProps.cs#L562-L586

Reply:
Thank you for the comment.
Regarding the positive test, I was wondering if `SetWindowSize_GetWindowSize_ReturnsExpected` actually tests `Console.SetWindowSize` on Windows test env.
On linux test env, `Console.IsOutputRedirected` is `true` and it makes whole test context skipped.
I can confirm that it works on linux when console output is not redirected.

Reply:
[5150fcc](https://github.com/dotnet/runtime/pull/75824/commits/5150fcc33970c26caab2731e2afa36d40a397297) is pushed for consistency with Windows

Reply:
@adamsitnik Please review this.